src/ColorVectorSpace.jl

```diff
@@ -8,7 +8,7 @@ import Base: ==, +, -, *, /, .+, .-, .*, ./, ^, <, ~
 import Base: abs, abs2, clamp, convert, copy, div, eps, isfinite, isinf,
     isnan, isless, length, mapreduce, norm, one, promote_array_type, promote_rule, zero,
     trunc, floor, round, ceil, bswap,
-    mod, rem, atan2, hypot, max, min, varm, real
+    mod, rem, atan2, hypot, max, min, histrange
```

Comment:
The Travis failures on this PR are a consequence of deleting the imports for `varm` and `real`. You need to add `histrange` to the list, keeping rather than replacing those.

If you don't know this already, you can always run the tests locally (`include("runtests.jl"` or `Pkg.test("ColorVectorSpace")`). I don't always remember to do that myself before pushing, but it can save a Travis iteration or two.